web/app/static/js/components/azurecredential/azurecredential.js

```diff
@@ -3,13 +3,19 @@ angular.module('cloudbreakApp').component('azureCredential', {
     controller: AzureCredentialController
 });
 
-AzureCredentialController.$inject = ['$rootScope', '$filter', 'InteractiveLogin', 'AccountCredential', 'UserCredential'];
+AzureCredentialController.$inject = ['$rootScope', '$filter', 'InteractiveLogin', 'AccountCredential', 'UserCredential', 'DefaultSsh'];
 
-function AzureCredentialController($rootScope, $filter, InteractiveLogin, AccountCredential, UserCredential) {
+function AzureCredentialController($rootScope, $filter, InteractiveLogin, AccountCredential, UserCredential, DefaultSsh) {
     var ctrl = this;
+    var defaultSshKey;
 
     ctrl.$onInit = function () {
-        init();
+        DefaultSsh.get(function (result) {
+            defaultSshKey = result.defaultSshKey;
+            init();
```

Comment:
I suggest to move init() one level out. As i see it doesn't have any dependency on default ssh key